wannierberri/readme.tmp

```diff
@@ -0,0 +1 @@
+kubo is called from integrapte.py, so shift current should also be called from there
```

Comment:
this file is not needed.

Reply:
Hi Stepan

I performed tests on GaAs, it looks much faster than postw90. I ran WB with one processor in my local machine, fine mesh of NKdiv=[7 7 7], NKFFT=[13 13 13], NKtot=[91 91 91] one Fermi energy, 200 frequencies, and it took 972 s. 

I ran postw90 in same 91x91x91 k-grid and same frequencies, in parallel with 8 processors, it took 1772 s. If Im not wrong, this means WB is more than one order of magnitude faster per processor. 

But it is possible that some of the np.einsum could be speed up by some other means.